### PR TITLE
HPCC-14385 Minor fixes in preparation for HPCC-9318

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday gavin.halliday@lexisnexis.com

### CODE REVIEW

**Comment by ghalliday** [comment]
@jamienoss please review

File: ecl/hql/hqlexpr.cpp [hunk 1]
```diff
@@ -6349,6 +6346,8 @@ void CHqlDataset::cacheParent()
     case no_mergejoin:
     case no_nwayjoin:
     case no_nwaymerge:
+    case no_serialize:
+    case no_deserialize:
         rootTable = this;       //?
```
Comment (jamienoss) [inline ecl/hql/hqlexpr.cpp]: @ghalliday whilst here might it be worth addressing this comment "//?" or at least expanding on it - or perhaps open another jira?

**Comment by jamienoss** [comment]
@richardkchapman other than the observation of a comment cleaning opportunity, this looks good to merge.

File: ecl/hql/hqlexpr.cpp (hunk 1 above)
Comment (richardkchapman) [inline ecl/hql/hqlexpr.cpp]: Just needs deleting really. I will do that as I merge.

**Pull request merged**